Makefile

```diff
@@ -50,10 +65,23 @@ behat.yml: behat.yml.dist
 		cp behat.yml.dist behat.yml;\
 	fi
 
-book:
-	#
-	# Darkmira Community Event Manager
-	#
-	# Application: http://0.0.0.0:8080
-	# Minio:       http://0.0.0.0:9001
-	#
+##
+## Tools
+## -----
+##
+
+logs: ## Show logs
+	docker-compose logs -ft
+
+bash: ## Bash into php container
```

Comment:
I'm quite against having people doing things inside the container to be fair...

Reply:
you mean you dont want make bash ?

Reply:
Yes indeed. I can't see any reasonable reason you'd need to get inside the container in a quick and efficient way. Having you type the full command gives you time to think about what mistake you're going to commit ;p

Reply:
ok for me i delete this one